src/features/list-view/__tests__/ListView-test.js

```diff
@@ -50,6 +53,26 @@ describe('features/list-view/ListView', () => {
 
             expect(testCell.hasClass(className)).toBeTruthy();
         });
+
+        test.each`
+            columnIndex | rowIndex | cellData                   | should                                                                                                        | className
+            ${0}        | ${0}     | ${'h1<IconSortChevron />'} | ${'returns h1<IconSortChevron /> when columnIndex is 0 and rowIndex is 0 and getGridHeaderSort is passed in'} | ${'.icon-sort-chevron.is-sort-asc'}
+            ${1}        | ${0}     | ${'h2'}                    | ${'returns h2 when columnIndex is 0 and rowIndex is 0 and getGridHeaderSort is passed in'}                    | ${'.icon-short-chevron'}
```

Comment:
'returns h2 when columnIndex is 1 and rowIndex is 0 and getGridHeaderSort is passed in'